src/pages/author/[...slug].tsx

```diff
@@ -3,21 +3,18 @@ import { NextSeo } from 'next-seo';
 import { ParsedUrlQuery } from 'querystring';
 import { getAllAuthors } from '../../service/get-all-authors';
 import { Author, Image } from '../../interfaces/author';
-import {
-    Button,
-    Eyebrow,
-    GridLayout,
-    SpeakerLockup,
-    TypographyScale,
-} from '@mdb/flora';
+import { Button, GridLayout, SpeakerLockup, TypographyScale } from '@mdb/flora';
 import React, { useState } from 'react';
 import Card, { getCardProps } from '../../components/card';
 import { getAllContentItems } from '../../service/get-all-content';
 import { ContentItem } from '../../interfaces/content-item';
 import { Grid } from 'theme-ui';
 import { getPlaceHolderImage } from '../../utils/get-place-holder-thumbnail';
 import AuthorSocialButtons from '../../components/author-social-buttons';
+import Breadcrumbs from '../../components/breadcrumbs';
+import { Crumb } from '../../components/breadcrumbs/types';
 
+const crumbs: Crumb[] = [{ text: 'MongoDB Developer Center', url: '/' }];
```

Comment:
So the authors page only has one breadcrumb?

Reply:
Yeah so this is what it is in the design, but it doesn't really make sense to have the Article Name part in there, so I just did the first one. https://devcenter-frontend.devrel.prod.corp.mongodb.com/developer/topics/ also has 1 breadcrumb so I think it should be ok.